articles/aks/availability-zones.md

```diff
@@ -85,7 +84,8 @@ If a single zone becomes unavailable, your applications continue to run if the c
 
 When you create a cluster using the [az aks create][az-aks-create] command, the `--zones` parameter defines which zones agent nodes are deployed into. The control plane components such as etcd or the API are spread across the available zones in the region if you define the `--zones` parameter at cluster creation time. The specific zones which the control plane components are spread across are independent of what explicit zones are selected for the initial node pool.
 
-If you don't define any zones for the default agent pool when you create an AKS cluster, control plane components are not guaranteed to spread across availability zones. You can add additional node pools using the [az aks nodepool add][az-aks-nodepool-add] command and specify `--zones` for new nodes, but it will not change how the control plane has been spread across zones. Availability zone settings can only be defined at cluster or node pool create-time.
+If you don't define any zones for the default agent pool when you create an AKS cluster, control plane will not be in Availability Zones.
```

Comment:
Suggest rewording to - If you don't define any zones for the default agent pool when you create an ASK cluster, the control plane components won't be present in availability zones.

Reply:
It is perfect for me. I'm not an English native speaker, feel free to change the text!! :)